pkg/controller/node/ipam/range_allocator.go

```diff
@@ -176,7 +176,10 @@ func (r *rangeAllocator) worker(stopChan <-chan struct{}) {
 				glog.Warning("Channel nodeCIDRUpdateChannel was unexpectedly closed")
 				return
 			}
-			r.updateCIDRAllocation(workItem)
+			if err := r.updateCIDRAllocation(workItem); err != nil {
+				// Requeue the failed node for update again.
+				r.insertNodeToProcessing(workItem.nodeName)
```

Comment:
The same comments as in cloud_cidr_allocator apply here.

Reply:
Done.